src/test/resources/jarWithCompileDependency/project2/pom.xml

```diff
@@ -36,6 +36,11 @@
 			<artifactId>jarWithCompileDependency1</artifactId>
 			<version>1.0</version>
 		</dependency>
+		<dependency>
+            <groupId>com.google.guava</groupId>
+            <artifactId>guava</artifactId>
+            <version>30.1.1-android</version>
```

Comment:
Why `-android` and not `-jre` (which seems to be more suitable)?

Reply:
This is a test. This is only a test. had this been a real dependency you'd get a different answer.

Reply:
Oh right, good point. I *did* just see a `-android` guava dependency in a Maven plugin (on mvnrepository.com), that’s why…